tests/unit/test_replicator_dynamics.py

```diff
@@ -0,0 +1,1207 @@
+"""
+Tests for Replicator Dynamics
+"""
+# import types
```

Comment:
If we don't need it, remove it. As we're using version control the idea of "commenting something out to keep it for later" is not necessary.

```suggestion
```